espflash/src/error.rs

```diff
@@ -44,6 +44,11 @@ pub enum Error {
     #[error(
         "Supplied ELF image of {0}B is too big, and doesn't fit configured app partition of {1}B"
     )]
+    #[diagnostic(
+        code(espflash::image_too_big),
+        help("Reduce the size of the binary or increase the size of the partition."),
```

Comment:
Would have preferred this to say "increase the size of the __app__ partition", as it's pretty ambiguous now. Will fix it in another PR before release I guess.